CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ vX.X.X - DD MMM YYYY
 --------------------
 
 Features:
+* [#433](https://github.com/godaddy/tartufo/pull/433) - Dropped support for deprecated flags from v3.0
```

Comment:
In this context, and in the interest of visibility, I would explicitly list the removed flags here again... for people who only read the CHANGELOG and not the error messages. ;-)

Reply:
+1